README.md

```diff
@@ -0,0 +1,216 @@
+
+### CarbonCulture's entities
+
+CarbonCulture's datastreams are divided into entity types, which consist of
+Communities, Places and Meters. A Community is a group of people, who may or
+may not exist in the same physical location, a Place is a physical entity
+around which you could draw a line (if you had enough chalk and lots of
+ladders). A Meter is really an energy data-source rather than necessarily
+being a true physical metering point, and could be derived from one, two (or
+more) input datastreams (e.g. real-time data and a backup half-hourly feed).
+
+You are able to request a section of data for a particular entity on the
+CarbonCulture platform, over a time range, at different resolutions, in
+different units, and with other helpful metadata -- depending on your access
+level and the availability of these data.
+
+In order to access these data, you will need to know the unique identifier (ID)
+of the entity, and this can be found in the URL of the page on
+carbonculture.net that represents this entity, for example UK Parliament (the
+Community) has an ID of 2:
+
+https://platform.carbonculture.net/communities/uk-parliament/2/
+
+Cardiff Castle's ID is 947:
+
+https://platform.carbonculture.net/places/cardiff-castle/947/
+
+Note that a Place or Meter may have the same ID as a Community, but Places
+and Meters never share IDs (this is because internally Places and Meters are
+both represented as the same data structure - a physical 'Asset').
+
+### Where/how to find data
+
+CarbonCulture's datastreams ('Channels') are named in a similar way to these
+URLs, so for example if you want to access UK Parliament's electricity data,
+you would need the Channel called `communities.2.elec`, while Cardiff Castle's
+gas data will be at `places.947.gas`.
+
+In order to discover what utilities are accessible to you for an entity, you
+can use the subject module, so let's look at an example of connecting and
+discovering:
+
+```
+In [1]: from xylem.connection import Connection
+
+In [2]: xc = Connection(
+                'carbonculture@apps.utility-graph?communities.2',
+                'bcac16a7f1fc2e32c9e41ea4ff9c7d693e49714e')
+
+In [3]: from xylem.subjects import discover_available_resources
+
+In [4]: discover_available_resources(xc, 2, 'communities')
+Out[4]:
+{u'elec': {u'data_type': u'',
+  u'description': u'All electricity used by the UK Parliament community',
+  u'earliest_timestamp': u'2013-07-17T00:00:00',
+  u'gui_max': 5500.0,
+  u'latest_timestamp': u'2015-05-12T00:00:00',
+  u'licence': u'None',
+  u'meta': u'',
+  u'permissions': [u'read_history_full'],
+  u'quality_assurance': [],
+  u'resource_uri': u'/api/v1/channel/communities.2.elec',
+  u'slug': u'communities.2.elec',
+  u'stats': None,
+  u'store_history': False,
+  u'unit': u'kWh',
+  u'value_type': u'accum',
+  u'values': []},
+
+...
+
+```
+
+There's quite a lot of info in this result, but the pertinent points are the
+result keys:
+
+```
+In [5]: _4.keys()
```

Comment:
might be slightly clearer to manually assign the result to a variable and call that, just for clarity?